src/Mailgun/Model/MailingList/MailingList.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+/*
+ * Copyright (C) 2013 Mailgun
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+
+namespace Mailgun\Model\MailingList;
+
+/**
+ * @author Michael Münch <helmchen@sounds-like.me>
+ */
+final class MailingList
```

Comment:
Due this modal is part of the `Response` and given the fact that we need to implement the `create` static method, is not a bad idea if you add the `implements ApiResponse` sentence here. In this way we will have an uniform format for all files that are returned as part of any request.

Same case for other models should be implemented too please. Thanks!

Reply:
You're right. But maybe an ApiModel interface would be more useful here in the future? Because, strictly speaking, it's actually a model, not a response itself, but that's just conceptual

Reply:
That makes sense to me, however I believe it's not a good idea to add a new `ApiModel` to define the same static method, so I believe that a new interface extending a previous one would be better, so in the end both `ApiResponse` and `ApiModel` might extend from a parent interface that might define the static method.